contracts/cw1-subkeys/Cargo.toml

```diff
@@ -20,12 +20,12 @@ test-utils = []
 
 [dependencies]
 cosmwasm-schema = { version = "1.1.0" }
-cw-utils = "0.16.0"
+cw-utils = "1.0.1"
 cw1 = { path = "../../packages/cw1", version = "1.0.0" }
 cw2 = { path = "../../packages/cw2", version = "1.0.0" }
 cw1-whitelist = { path = "../cw1-whitelist", version = "1.0.0", features = ["library"] }
 cosmwasm-std = { version = "1.1.0", features = ["staking"] }
-cw-storage-plus = "0.16.0"
+cw-storage-plus = "1"
```

Comment:
Let's use 1.0.1 here as well since 1.0.0 is known to be broken for Wasm

Reply:
Hmm strange. For some reason the first time I tried building with `-Z minimal-versions` it worked, but now that I tried again it does indeed fail because cw-storage-plus 1.0.0 is selected. I must have missed something the first time. Will change to 1.0.1. Could be a good idea to also yank 1.0.0.

Reply:
When you use `-Z minimal-versions` you need to remove the lockfile fist. Otherwise the version from the lockfile is used.

> Could be a good idea to also yank 1.0.0.

This would not have helped here. A version that is in a lockfile already will still be used when yanked.